taxi_service/settings.py

```diff
@@ -143,3 +144,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK="bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice. You should remove the duplicate definition on this line to avoid redundancy.